packages/fxa-payments-server/src/routes/Checkout/index.tsx

```diff
@@ -185,7 +185,6 @@ export const Checkout = ({
             setSubscriptionError({ type: 'card_error', code: 'card_declined' });
           },
         });
-        Amplitude.createSubscriptionWithPaymentMethod_FULFILLED(selectedPlan);
```

Comment:
😓   Also, where's the other metrics call?

Reply:
It looks like most of the `_PENDING/_FULFILLED/_REJECTED` calls are wrapped around the API call in `apiClient`.

So in this case ` apiClient.ts > apiCreateSubscriptionWithPaymentMethod`